pkg/api/v1alpha1/cluster_types.go

```diff
@@ -762,6 +766,126 @@ type KindAccessor interface {
 	ExpectedKind() string
 }
 
+// PackageConfiguration for installing EKS Anywhere curated packages.
+type PackageConfiguration struct {
+	// Disable package controller on cluster
+	Disable bool `json:"disable,omitempty"`
+
+	// Controller package controller configuration
+	Controller PackageControllerConfiguration `json:"controller,omitempty"`
+
+	// Cronjob for ecr token refresher
+	CronJob PackageControllerCronJob `json:"cronjob,omitempty"`
+}
+
+// Equal for PackageConfiguration.
+func (n *PackageConfiguration) Equal(o *PackageConfiguration) bool {
+	if n == o {
+		return true
+	}
+	if n == nil || o == nil {
+		return false
+	}
+	return n.Disable == o.Disable && n.Controller.Equal(&o.Controller)
+}
+
+// PackageControllerConfiguration configure aspects of package controller.
+type PackageControllerConfiguration struct {
+	// Repository package controller repository
+	Repository string `json:"repository,omitempty"`
+
+	// Tag package controller tag
+	Tag string `json:"tag,omitempty"`
+
+	// Digest package controller digest
+	Digest string `json:"digest,omitempty"`
+
+	// DisableWebhooks on package controller
+	DisableWebhooks bool `json:"disableWebhooks,omitempty"`
+
+	// Env of package controller
```

Comment:
could we document the format? I'm assuming `key=val`?

Reply:
Done.